book/src/reference/config.md

```diff
@@ -791,6 +791,17 @@ The supported choices are:
 By default all runs of `cargo-dist` will be trying to handle all platforms specified here at once. If you specify `--target=...` on the CLI this will focus the run to only those platforms. As discussed in [concepts][], this cannot be used to specify platforms that are not listed in `metadata.dist`, to ensure different runs agree on the maximum set of platforms.
 
 
+### vendor-workflow-deps
+
+> since 0.15.0
+
+Example: `vendor-workflow-deps = true`
+
+When enabled, `cargo dist init` will copy the complete source of (almost) all external GitHub actions into your git repository at `.github/actions` and configure your release workflow to use the local copies instead of the external copies. For most users, this is unnecessary; we expect this to only be of interest to users with specific restricted security contexts who want to ensure they're using specific known versions of actions without fetching them from third party repositories.
```

Comment:
same as above, can we wordsmith this to be more positive. i think a lot of our users are security conscious and so i think we should frame that as more normal/good